src/addons/model.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * @fileoverview A general modelling class used by the Presenter's
+ * getModel function.
+ */
+
+import { set, Emitter } from '../microcosm'
+
+function isObservable(binding) {
+  return binding && typeof binding.subscribe === 'function'
+}
+
+function isCallable(binding) {
+  return binding && typeof binding.call === 'function'
+}
+
+function invoke(binding, repo, scope) {
+  if (isCallable(binding)) {
+    return binding.call(scope, repo.state, repo)
+  }
+
+  return binding
+}
+
+export default class Model extends Emitter {
+  /**
+   * @param {Microcosm} repo Track this Microcosm instance for updates
+   * @param {scope} scope Scope to invoke functional bindings
+   */
+  constructor(repo, scope) {
+    super()
+
+    this.repo = repo
+    this.scope = scope
+    this.bindings = {}
+    this.subscriptions = {}
+    this.value = {}
+
+    this.repo.on('change', this.compute, this)
+  }
+
+  /**
+   * Track an observable. Sending updates to a given key.
+   * @param {string} key
+   * @param {Observable} observable
+   */
+  track(key, observable) {
+    let last = this.subscriptions[key]
+    let next = observable.subscribe(value => this.set(key, value))
+
+    this.subscriptions[key] = next
+
+    if (last) {
+      last.unsubscribe()
+    }
+  }
+
+  /**
+   * @param {Object} bindings An set of key/value pairs for building a model
+   */
+  bind(bindings) {
+    this.bindings = {}
+
+    for (var key in bindings) {
+      let binding = bindings[key]
+
+      if (isObservable(binding)) {
+        this.track(key, binding)
+      } else {
+        this.bindings[key] = binding
+      }
+    }
+
+    this.compute()
+  }
```

Comment:
Thanks a lot for clearing this up for me. Given the context I think `bind` is suitably accurate.